packages/datetime/src/dateInput.tsx

```diff
@@ -212,7 +212,14 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         const dateValue = this.isMomentValidAndInRange(value) ? fromMomentToDate(value) : null;
         const dayPickerProps = {
             ...this.props.dayPickerProps,
-            onMonthChange: () => setTimeout(this.registerPopoverBlurHandler, 0),
+            // dom elements for the updated month is not available when
+            // onMonthChange is called. setTimeout is necessary to wait
+            // for the updated month to be rendered
+            onMonthChange: (month: Date) =>
+                setTimeout(() => {
```

Comment:
please use `this.setTimeout` instead, and remove unnecessary `0` arg. see https://github.com/palantir/blueprint/blob/develop/packages/core/src/common/abstractComponent.ts#L43

Reply:
- [ ] use `this.setTimeout` remove `0`